app/controllers/cms/preview_controller.rb

```diff
@@ -64,143 +64,143 @@ def form_preview
   end
 
   private
-    def set_site
-      @cur_site = request.env["ss.site"] = SS::Site.find(params[:site])
-      @preview  = true
-    end
+  def set_site
+    @cur_site = request.env["ss.site"] = SS::Site.find(params[:site])
+    @preview  = true
+  end
 
-    def set_path_with_preview
-      set_site
-      @cur_path ||= request_path
-      @cur_path.sub!(/^#{cms_preview_path}(\d+)?/, "")
-      @cur_path = "index.html" if @cur_path.blank?
-      @cur_path = URI.decode(@cur_path)
-      set_main_path
-      @cur_date = params[:preview_date].present? ? params[:preview_date].in_time_zone : Time.zone.now
-      filters << :preview
-    end
+  def set_path_with_preview
+    set_site
+    @cur_path ||= request_path
+    @cur_path.sub!(/^#{cms_preview_path}(\d+)?/, "")
+    @cur_path = "index.html" if @cur_path.blank?
+    @cur_path = URI.decode(@cur_path)
+    set_main_path
+    @cur_date = params[:preview_date].present? ? params[:preview_date].in_time_zone : Time.zone.now
+    filters << :preview
+  end
 
-    def x_sendfile(file = @file)
-      return if file =~ /\.(ht|x)ml$/
-      return if file =~ /\.part\.json$/
-      super
-      return if response.body.present?
-
-      return unless @cur_path =~ /^\/fs\//
-      fs_path  = SS::Application.routes.recognize_path(@cur_path)
-      id_path  = fs_path[:id_path] || fs_path[:id]
-      action   = fs_path[:action]
-      size     = fs_path[:size]
-      filename = fs_path[:filename]
-      width    = params[:width]
-      height   = params[:height]
-
-      @item = SS::File.find_by(id: id_path.delete("/"), filename: filename) rescue nil
-      raise "404" unless @item
-      if width.present? && height.present?
+  def x_sendfile(file = @file)
```

Comment:
__[rubocop]__
* Cyclomatic complexity for x_sendfile is too high. [15/14]
* Perceived complexity for x_sendfile is too high. [17/14]

<!-- Comment by [SideCI](https://sideci.com) -->